v01/converter_test.go

```diff
@@ -363,13 +376,13 @@ func TestHTTPMarshallerFromRequestBinarySuccess(t *testing.T) {
 	factory := v01.NewDefaultHTTPMarshaller()
 
 	header := http.Header{}
-	header.Set("Content-Type", "application/json")
-	header.Set("CE-EventType", "com.example.someevent")
-	header.Set("CE-Source", "/mycontext/subcontext")
-	header.Set("CE-EventID", "1234-1234-1234")
-	header.Set("CE-MyExtension", "myvalue")
-	header.Set("CE-AnotherExtension", "anothervalue")
-	header.Set("CE-EventTime", "2018-04-05T03:56:24Z")
+	header.Set("content-type", "application/json")
+	header.Set("ce-eventtype", "com.example.someevent")
+	header.Set("ce-source", "/mycontext/subcontext")
+	header.Set("ce-eventid", "1234-1234-1234")
+	header.Set("ce-myextension", "myvalue")
```

Comment:
extensions are handled different, e.g.:

```
CURL -X POST example/webhook.json \
-H 'traceparent:  00-4bf92f3577b34da6a3ce929d0e0e4736-00f067aa0ba902b7-01' \
-H 'tracestate: rojo=00-4bf92f3577b34da6a3ce929d0e0e4736-00f067aa0ba902b7-01,congo=lZWRzIHRoNhcm5hbCBwbGVhc3VyZS4=` \
-H 'content-type: application/cloudevents+json' \
-d '@sample-event.json'
```

for Distributed Tracing. See: https://github.com/cloudevents/spec/blob/master/extensions/distributed-tracing.md

Reply:
wrong section - sorry